### DISCUSSION TITLE
Running on Linux server, accessing from Windows browser.

### DISCUSSION THREAD

**Discussion by phalexo** [lllyasviel/stable-diffusion-webui-forge#394] [Q&A]
webui.py attempts to open a browser on the Ubuntu server.

when I run

python webui.py

I just want it to put up an API on 0.0.0.0 <some port>, so I can access it from windows. I think that is how it used to work with the old branch.

Thanks.

(SDWebUI) developer@ai:~/stable-diffusion-webui$ python webui.py
Total VRAM 12213 MB, total RAM 322407 MB
xformers version: 0.0.25+158f36c.d20240224
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce GTX TITAN X : native
VAE dtype: torch.float32
CUDA Stream Activated:  False
Using xformers cross attention
ControlNet preprocessor location: /home/developer/stable-diffusion-webui/models/ControlNetPreprocessor
2024-02-24 19:42:05,403 - roop - INFO - roop v0.0.2
2024-02-24 19:42:05,472 - roop - INFO - roop v0.0.2
Delete Button: send2trash is not installed. recycle bin cannot be used.
Image Browser: send2trash is not installed. recycle bin cannot be used.
Loading weights [31e35c80fc] from /home/developer/stable-diffusion-webui/models/Stable-diffusion/sd_xl_base_1.0.safetensors
2024-02-24 19:42:07,121 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 16.9s (import torch: 6.5s, import gradio: 0.8s, setup paths: 1.0s, other imports: 0.6s, load scripts: 2.8s, create ui: 4.5s, gradio launch: 0.7s).
/usr/bin/xdg-open: 869: x-www-browser: Permission denied
/usr/bin/xdg-open: 869: firefox: Permission denied
/usr/bin/xdg-open: 869: iceweasel: Permission denied
/usr/bin/xdg-open: 869: seamonkey: Permission denied
/usr/bin/xdg-open: 869: mozilla: Permission denied
/usr/bin/xdg-open: 869: epiphany: Permission denied
/usr/bin/xdg-open: 869: konqueror: Permission denied
/usr/bin/xdg-open: 869: chromium: Permission denied
/usr/bin/xdg-open: 869: chromium-browser: Permission denied
/usr/bin/xdg-open: 869: google-chrome: Permission denied
/usr/bin/xdg-open: 869: www-browser: Permission denied
/usr/bin/xdg-open: 869: links2: Permission denied
/usr/bin/xdg-open: 869: elinks: Permission denied
/usr/bin/xdg-open: 869: links: Permission denied
/usr/bin/xdg-open: 869: lynx: Permission denied
/usr/bin/xdg-open: 869: w3m: Permission denied
xdg-open: no method available for opening 'http://127.0.0.1:7860/'
model_type EPS
UNet ADM Dimension 2816
Using xformers attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using xformers attention in VAE
extra {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
Loading VAE weights specified in settings: /home/developer/stable-diffusion-webui/models/VAE/sdxl_vae.safetensors
To load target model SDXLClipModel
Begin to load 1 model
Moving model(s) has taken 0.02 seconds
Model loaded in 20.8s (load weights from disk: 3.1s, forge load real models: 14.7s, load VAE: 0.3s, calculate empty prompt: 2.6s)

**Comment by salocinmad** [comment]
first run with correct parameters, in this case webui.sh --listen
And check all files are with correct username assigmend... in dont know you can do it
sudo chown username.username /folder-stable-diffusion
example
sudo chown salocin.salocin /workspace/stable-diffusion-webui-forge
